src/Aggregation/BucketScript.php

```diff
@@ -71,7 +71,8 @@ public function toArray(): array
     {
         if (!$this->hasParam('buckets_path')) {
             throw new InvalidException('Buckets path is required');
-        } elseif (!$this->hasParam('script')) {
```

Comment:
Was this converted automatically? I think both options will work but they are not 100% identical I would argue.

Reply:
yes, all the changes were done by the cs-fixer

Reply:
well, it is identical. as the if branch is throwing an exception :)

Reply:
damn, true :-D `thePanz 1 : 0 ruflin`